content/cloud/contributing.md

```diff
@@ -60,24 +63,66 @@ Create a new remote for the upstream (a pointer to the original repository to wh
 $ git remote add upstream https://github.com/fermyon/developer
 ```
 
-## 5. Add Changes
+## 5. Code Blocks and Annotations
+
+It is highly recommended that you use either the `<!-- @selectiveCpy -->` or the `<!-- @nocpy -->` annotation before each of your code blocks, and that each code block defines the appropriate [syntax highlighting](https://rdmd.readme.io/docs/code-blocks#language-support). 
```

Comment:
The annotation can be skipped for code blocks with example code snippets (i.e) non terminal/output type codeblocks

Reply:
Also, updated in latest commit.